ui/components/ui/form-field/form-field.js

```diff
@@ -115,6 +117,7 @@ export default function FormField({
             className={classNames('form-field__input', {
               'form-field__input--error': error,
               'form-field__input--warning': warning,
+              'form-field__input--colour-value': coloredValue,
```

Comment:
Nit: We've been using the America-centric `color` instead

Reply:
Thank you @darkwing for fixing this color spelling! I appreciate it! 😄